packages/stockflux-watchlist/public/preview-chart.html

```diff
@@ -12,6 +12,9 @@
         box-sizing: border-box;
         width: 100vw;
         height: 100vh;
+        background-color:#1B3B56;
```

Comment:
Assuming it's not possible to make this into a `var` since it's a public/.html file?

Reply:
That's what i thought initially, but upon changing them to use the var from the main index file it could reference them, so changed to use the var values.

Reply:
Is this not using `useChildWindow`? This would allow CSS vars to actually be rendered to appropriate values.